test_teardown.splitbrain.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- hosts: usm_client[0]
+  remote_user: root
+  tasks:
+    - name: Remove splitbrain invoking iptables rules
+      iptables:
+        action: insert
+        state: absent
+        chain: OUTPUT
+        destination: "{{ item }}"
+        jump: DROP
+      with_items: "{{ groups['gluster_splitbrain0'] }}"
+
+- hosts: usm_client[1]
+  remote_user: root
+  vars:
+    volume_mountpoint: "/mnt/volume_tmp_splitbrain"
+    volume_name: "volume_alpha_distrep_6x2"
+  tasks:
+
+    - name: Remove splitbrain invoking iptables rules
+      iptables:
+        action: insert
+        state: absent
+        chain: OUTPUT
+        destination: "{{ item }}"
+        jump: DROP
+      with_items: "{{ groups['gluster_splitbrain1'] }}"
+
+    - name: Unmount GlusterSF volume for splitbrain purposes
+      mount:
+        path: "{{ volume_mountpoint }}"
+        state: unmounted
+
+    - name: Remove GlusterSF volume for splitbrain purposes from fstab
+      mount:
+        path: "{{ volume_mountpoint }}"
+        fstype: glusterfs
+        src: "{{ groups['gluster'][0] }}:{{ volume_name }}"
+        state: absent
+
+    - name: Remove splitbrain mountpoint directory
+      file:
+        path: "{{ volume_mountpoint }}"
+        state: absent
```

Comment:
Just curious if this is needed?

From [ansible doc](http://docs.ansible.com/ansible/latest/mount_module.html):
> Similarly, specifying `absent` will remove the mount point directory.

Reply:
You are right, it is not needed.

Reply:
Fixed in 7cf0ddcb4afc264f1c03af80b7b120b5eed81bd2 and tested that it works.